### PR TITLE
ticdc: add more faqs

### PR DESCRIPTION
**Pull request by ran-huang**
Signed-off-by: Ran <huangran@pingcap.com>

### What is changed, added or deleted? (Required)

- Add more items in faq
- Add Ordering guarantees and transaction guarantees
- Fix some errors in command sample

### Which TiDB version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [x] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from: https://github.com/pingcap/docs-cn/pull/3723, https://github.com/pingcap/docs-cn/pull/3882
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Have version specific changes 
- [ ] Might cause conflicts

### CODE REVIEW

**Comment by TomShawn** [comment]
LGTM

File: ticdc/troubleshoot-ticdc.md [hunk 1]
```diff
@@ -55,3 +55,185 @@ cdc cli changefeed create --pd=http://10.0.10.25:2379 --start-ts=415238226621235
...
+Warning: Unable to load '/usr/share/zoneinfo/zone1970.tab' as time zone. Skipping it.
+```
+
+If you use MySQL in a special cloud environment, such Aliyun RDS, and if you do not have the permission to modify MySQL, you need to specify the time zone using the `--tz` parameter:
+
+1. Query the time zone used by MySQL:
+
+    {{< copyable "shell-regular" >}}
+
+    ```shell
```
[the remark this answers is not in the document]
Comment (amyangfei) [reply ticdc/troubleshoot-ticdc.md:83]: should use `sql`

File: ticdc/troubleshoot-ticdc.md [hunk 2]
```diff
@@ -55,3 +55,185 @@ cdc cli changefeed create --pd=http://10.0.10.25:2379 --start-ts=415238226621235
...
+Warning: Unable to load '/usr/share/zoneinfo/zone1970.tab' as time zone. Skipping it.
+```
+
+If you use MySQL in a special public cloud environment, such Alibaba Cloud RDS, and if you do not have the permission to modify MySQL, you need to specify the time zone using the `--tz` parameter:
+
+1. Query the time zone used by MySQL:
+
+    {{< copyable "shell-regular" >}}
+
+    ```shell
```
Comment (amyangfei) [inline ticdc/troubleshoot-ticdc.md:83]:
```suggestion
    ```sql
```

File: ticdc/troubleshoot-ticdc.md [hunk 3]
```diff
@@ -55,3 +55,185 @@ cdc cli changefeed create --pd=http://10.0.10.25:2379 --start-ts=415238226621235
...
+Warning: Unable to load '/usr/share/zoneinfo/leap-seconds.list' as time zone. Skipping it.
+Warning: Unable to load '/usr/share/zoneinfo/zone.tab' as time zone. Skipping it.
+Warning: Unable to load '/usr/share/zoneinfo/zone1970.tab' as time zone. Skipping it.
+```
+
+If you use MySQL in a special public cloud environment, such Alibaba Cloud RDS, and if you do not have the permission to modify MySQL, you need to specify the time zone using the `--tz` parameter:
+
+1. Query the time zone used by MySQL:
+
+    {{< copyable "shell-regular" >}}
```
Comment (TomShawn) [inline ticdc/troubleshoot-ticdc.md:81]:
```suggestion
    {{< copyable "sql" >}}
```

**Pull request merged**